File: python/post.py
import requests
import json

def main():
    protocol = "https"
    domain = "new.beluga.fm"
    api_version = "v1"
    access_token = "your_access_token"
    access_token_secret = "your_access_token_secret"

    url = "{}://{}/api/{}/status/update".format(protocol, domain, api_version)
    headers = {
        "Content-Type": "application/json"
    }
    payload = {
        "access_token": access_token,
        "access_token_secret": access_token_secret,
        "channel_id": "5ac0cb58b806cf47f01ba0db",
        "text": "ふー"
    }
    response = requests.post(url, headers=headers, data=json.dumps(payload))
    print(response)

if __name__ == "__main__":
    main()

File: node/examples/post.js
// axiosの準備
const axios = require("axios")
const protocol = "https"
const domain = "new.beluga.fm"
const api_version = "v1"
const request = axios.create({
	"baseURL": `${protocol}://${domain}/api/${api_version}/`,
	"headers": {
		"Content-Type": "application/json"
	},
	"responseType": "json"
})

// APIキー
const access_token = "your_access_token"
const access_token_secret = "your_access_token_secret"

// 投稿先
const hashtag_id = "5a8da737f734407078dd9583"
const text = "ふー"

// 投稿する
request
	.post("/status/update", {
		access_token,
		access_token_secret,
		hashtag_id,
		text
	})
	.then(res => {
		const { data } = res
		if (data.success == false) {
			console.error(data.error)
			return
		}
		console.log(data)
	})
	.catch(error => {
		console.error(error)
	})